src/clp_ffi_py/ir/native/PyFourByteSerializer.cpp

```diff
@@ -0,0 +1,143 @@
+#include <clp_ffi_py/Python.hpp>  // Must always be included before any other header files
+
+#include "PyFourByteSerializer.hpp"
+
+#include <clp_ffi_py/ir/native/serialization_methods.hpp>
+#include <clp_ffi_py/PyObjectCast.hpp>
+#include <clp_ffi_py/utils.hpp>
+
+namespace clp_ffi_py::ir::native {
+namespace {
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays)
+PyDoc_STRVAR(
+        cSerializePreambleDoc,
+        "serialize_preamble(ref_timestamp, timestamp_format, timezone)\n"
+        "--\n\n"
+        "Serializes CLP preamble for a 4-byte encoded CLP IR stream.\n\n"
```

Comment:
```suggestion
        "Serializes the preamble for a 4-byte encoded CLP IR stream.\n\n"
```
Alternatively: `Serializes the CLP IR preamble ...`